lib/src/gotrue_api.dart

```diff
@@ -139,6 +141,66 @@ class GoTrueApi {
     }
   }
 
+  /// Logs in an existing user using an id token from a provider.
+  ///
+  /// `idToken` is the user id token with nonce
+  ///
+  /// `nonce` is the unhashed nonce in id token
+  ///
+  /// `provider` is the id provider. Can be `apple`, `google`, `azure` or
+  /// `facebook`
+  Future<GotrueSessionResponse> signInWithToken(
```

Comment:
I wonder if we should use the same method name as the Javascript SDK. Was there a particular reason why you changed the method name here?

https://github.com/supabase/gotrue-js/blob/master/src/GoTrueApi.ts#L227

Reply:
I didn't know there was a javascript function for this... https://github.com/supabase/gotrue-js/issues/169 is still open

Reply:
It was actually merged in. I think the issue is still just open